templates/solr.service.j2

```diff
@@ -6,10 +6,11 @@ After = syslog.target network.target remote-fs.target nss-lookup.target
 Type = forking
 User={{ dataverse.solr.user }}
 Group={{ dataverse.solr.group }}
-WorkingDirectory = {{ dataverse.solr.root }}/server/solr/collection1
```

Comment:
I'm a little confused why the "collection1" stuff is being removed from "WorkingDirectory". It is related to the "-j" stuff?

Reply:
While I was in the solr template I compared the role's copy against what IQSS recommends in the Dataverse docs, and doctored up the working directory and NPROC while I was in there.